src/components/Archive.jsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import {Box,Button, Typography} from '@mui/material';
+import IconButton from '@mui/material/IconButton';
+import { DeleteOutlineOutlined } from '@mui/icons-material';
```

Comment:
Consider using barrel imports here. Import all your icons into one file and use them as per your need. The code will look clean

Reply:
These are imports from the material ui. These components are not created by me. Hence not using barrel imports for these.